Tabloid/client/src/components/ApplicationViews.js

```diff
@@ -49,6 +50,10 @@ export default function ApplicationViews({ isLoggedIn }) {
           {isLoggedIn ? <TagForm /> : <Redirect to="/login" />}
         </Route>
 
+        <Route path="/tags/edit/:id" >
```

Comment:
should the id in the route path be an integer? if so, please add some regex to ensure this route path is only matched when it is an integer.

Reply:
Fixed